docs/products/kafka/howto/fake-sample-data.rst

```diff
@@ -0,0 +1,63 @@
+Fake sample dataset
+===================
+
+Learning streaming is way more fun with data, so to get you started on your Apache Kafka® journey we help you creating fake streaming data to a topic.
+
+.. Note::
+
+    The following example is based on `Docker <https://www.docker.com/>`_ images, which require `Docker <https://www.docker.com/>`_ or `Podman <https://podman.io/>`_ to be executed.
+
+The following example assumes you have an Aiven for Apache Kafka® service running. You can create one following the :doc:`dedicated istructions <../getting-started>`.
+
+
+Fake data generator on Docker
+-----------------------------
+
+To learn data streaming, you need a continuous flow of data and for that you can use the `Dockerized fake data producer for Aiven for Apache Kafka® <https://github.com/aiven/fake-data-producer-for-apache-kafka-docker>`_. To take the generator into use:
```

Comment:
Maybe `To start using the generator` rather than `To take the generator into use`?

(I think I already made a similar change further down)